src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__staff_renewal_feed.sql

```diff
@@ -35,92 +35,237 @@ with
+
+            ayl.abbreviation as ay_school_shortname,
+
+            nyl.abbreviation as ny_school_shortname,
+
+            ayc.name as ay_campus_name,
+
+            nyc.name as ny_campus_name,
+
+            ayhos.sam_account_name as ay_head_of_school_samaccount,
+
+            nyhos.sam_account_name as ny_head_of_school_samaccount,
+
+            concat(b.family_name_1, ', ', b.given_name) as preferred_name,
+            concat(m.family_name_1, ', ', m.given_name) as manager_name,
+
+            case
+                when s.seat_tracker_id_number is null
+                then
+                    concat(
+                        'No Seat (',
+                        coalesce(s.nonrenewal_reason, 'No Reason Given'),
+                        ')'
+                    )
+                when
+                    (
+                        count(s.seat_tracker_id_number) over (
+                            partition by s.employee_number, s.academic_year
+                        )
+                    )
+                    > 1
+                then 'Is in multiple seats. Please update Seat Tracker❗'
+                when
+                    s.seat_tracker_id_number is not null
+                    and s.ay_business_unit != s.ny_entity
+                then 'Entity Changer - New Offer Letter'
+                when
+                    s.seat_tracker_id_number is not null
+                    and s.ay_location != s.ny_location
+                then 'Renew - Location Change'
+                when s.seat_tracker_id_number is not null
+                then 'Renew'
+                else 'Seat Tracker Error'
+            end as ny_status,
+
+        from {{ ref("int_people__staff_roster") }} as b
+        left join
+            {{ ref("int_people__staff_roster") }} as m
+            on b.reports_to_employee_number = m.employee_number
+        left join
+            {{ ref("int_people__renewal_status") }} as s
+            on b.employee_number = s.employee_number
+            and s.academic_year = {{ var("current_academic_year") }}
+        left join
+            {{ ref("int_surveys__renewal_responses_feed") }} as rf
+            on s.academic_year = rf.campaign_academic_year
+            and b.employee_number = rf.subject_employee_number
+            and rf.rn_approval = 1
+        left join
+            approvals as ap
+            on s.academic_year = ap.campaign_academic_year
+            and b.employee_number = ap.subject_employee_number
+        left join
+            {{ ref("stg_people__location_crosswalk") }} as ayl
+            on s.ay_location = ayl.name
+        left join
+            {{ ref("stg_people__location_crosswalk") }} as nyl
+            on s.ny_location = nyl.name
+        left join
+            {{ ref("stg_people__campus_crosswalk") }} as ayc
+            on ayl.clean_name = ayc.location_name
+        left join
+            {{ ref("stg_people__campus_crosswalk") }} as nyc
+            on nyl.clean_name = nyc.location_name

```

Comment:
therse should prob be refactored to `int_people__renewal_status`

Reply:
fair. I'll add it there